ldk/javascript/src/webpack/CheckWarningsAndErrorsPlugin.ts

```diff
@@ -16,6 +16,12 @@ export class CheckLoopOpenHandlerPlugin {
             openHandlerMissingWarning.file = assetName;
             compilation.warnings.push(openHandlerMissingWarning);
           }
+          if (fileContent?.includes('screen.ocr(')) {
+            const ocrDeprecationWarningMsg =
+              '[screen.ocr] ocr will be deprecated soon. Please use listenOcrMonitor instead.';
+            const openHandlerMissingWarning = new WebpackError(ocrDeprecationWarningMsg);
```

Comment:
Maybe rename this to `ocrDeprecationWarning`

Reply:
Thank you Jeff!!! I just updated the name. Please take a look! :)